src/index.ts

```diff
@@ -102,8 +114,9 @@ export default class IncludeFragmentElement extends HTMLElement {
     })
   }
 
+  // TODO: Should this return `this.#getData()` directly?
```

Comment:
I could use advice on this. On the one hand, it would be less surprising (and backwards-compatible) to always return a string.

However, that string can't be assigned to another `.innerHTML` without going through a CSP trusted types policy again.

Reply:
I could still use thoughts on this.